content/en/core/releases/4.4.0.md

```diff
@@ -52,9 +52,32 @@ As part of this work, some translation keys changed. We have provided translatio
 
 ## Highlights
 
-### Store purged docs cache in couchdb
+### Upgrading CouchDb to v3.3.2
 
-Each user gets their own doc in a new database `medic-cache` to store purged document ids the database is destroyed (deleted and recreated) every time purging runs
+CouchDb 3 has been released in 2020. It packs security fixes, performance and scalability improvements and a plethora of bug fixes.
+
+
+Most notable features are:
+1. Improved clustering and node management APIs
+2. Adding the `_reshard` endpoint, which allows splitting a shard into two shards. In CouchDb v2, adding additional shards was not possible without creating a new database with the desired number of shards, and replicating existent data to the new database. For the CHT, this meant that all users would need to re-sync if additional shards were added.
+3. Adding a new `_prometheus` endpoint, which exposes CouchDb metrics and can be used out of the box in monitoring visualisation tools, like the CHT Watchdog.
+4. Improved security, where database creation and writes/reads are no longer allowed for all users by default.
+5. Improved security, where only CouchDb admins are allowed to add/edit/delete users.
+6. Dramatically improved multi-core CPU usage, resulting in better performance when running on powerful machines. This results in a 1.5x improvement of request time across the board, with some endpoints being up to 10x faster.
+
+### Replication performance improvement by persistent caching of purged doc ids
+
+CouchDb request performance analysis has revealed that approximately 25% of CPU time was used querying to determine which documents have been purged for users that are replicating.
+Because purging is an action that runs on a schedule, relatively infrequently due to how performance expensive it is, caching results from these very expensive queries significantly improve server-side performance.
```

Comment:
```suggestion
Because purging is an action that runs on a schedule, relatively infrequently due to how performance expensive it is, caching results from these very expensive queries significantly improves server-side performance.
```